content/docs/command-reference/fetch.md

```diff
@@ -11,19 +11,28 @@ usage: dvc fetch [-h] [-q | -v] [-j <number>] [-r <name>] [-a] [-T]
                  [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-The `dvc fetch` downloads DVC-tracked files from remote storage into the cache
-of the project, but without placing them in the <abbr>workspace</abbr>. This
-makes the data files available for linking (or copying) into the workspace.
-(Refer to [dvc config cache.type](/doc/command-reference/config#cache).) Along
-with `dvc checkout`, it's performed automatically by `dvc pull` when the target
-`dvc.yaml` or `.dvc` files are not already in the cache:
+Downloads DVC-tracked files from remote storage into the cache of the project
+(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
+This makes them available for linking (or copying) into the workspace (refer to
+[`dvc config cache.type`](/doc/command-reference/config#cache)).
+
+Without arguments, `dvc fetch` ensures that the files specified in all
+`dvc.lock` and `.dvc` files in the workspace exist in the cache. The
+`--all-branches`, `--all-tags`, and `--all-commits` options enable fetching data
+for multiple workspace versions.
```

Comment:
`for multiple Git commits` is probably better ?

Reply:
How about "for `dvc.lock`/`.dvc` files in multiple Git commits"? Changed in d6f96108.

Reply:
Actually went for your text in the end (both here and in status) to simplify the language.